### PR TITLE
bump aiger version and support arbitrary initialization functions in btor2aiger

### PR DESCRIPTION
**Pull request by gipsyh**
Actually, aiger supports arbitrary initialization functions

### CODE REVIEW

File: src/btor2aiger.cpp [hunk 1]
```diff
@@ -414,16 +414,11 @@ add_state_to_aiger (Btor *btor,
...
-      /* Note: BTOR2 supports arbitrary initialization functions, but AIGER
-       * only supports 0/1/undefined. */
-      die ("Found non-constant initialization");
-    }
     sym = boolector_aig_get_symbol (amgr, state_bits[i]);
     if (next_bits)
     {
-      reset_val = init_bits ? init_bits[i] : state_bits[i];
+      reset_val = init_bits ? (init_bits[i] > 1 ? state_bits[i] : init_bits[i])
+                            : state_bits[i];
```
Comment (mpreiner) [inline src/btor2aiger.cpp:421]:
```suggestion
      reset_val = init_bits && init_bits[i] <= 1 ? init_bits[i] : state_bits[i];
```

**Comment by mpreiner** [comment]
@gipsyh Can you squash+sign your commits? Thanks!

**Comment by gipsyh** [comment]
Sorry, I didn't notice the message. I pulled the latest commits and squash+signed my commits, thanks.

**Comment by gipsyh** [comment]
Sorry, I found issues with the modifications I made, so I have withdrawn the pull request.

**Pull request closed without merging**